loops-and-arrays.js

```diff
@@ -41,6 +55,15 @@ const numbers = [2, 4, 5, 29, 38];
 // ==========================================
 
 const squares = [30, 2, 8, 24, 11];
+console.log(squares);
+
+for (let i = 0; i < squares.length; i++) {
+    const volume = squares[i] * squares[i] * squares[i];
```

Comment:
Machten in JavaScript kan je doen met de exponant operator `**` bijvoorbeeld `const volume = squares ** 3`